com.inochi2d.inochi-session.yml

```diff
@@ -0,0 +1,67 @@
+id: com.inochi2d.inochi-session
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ldc
+  - org.freedesktop.Sdk.Extension.llvm14
+command: inochi-session
+finish-args:
+  - --device=all          # OpenGL rendering, webcams (may be required? not sure)
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network       # Needed to communicate with external trackers
+  - --socket=wayland
+
+modules:
+  - shared-modules/SDL2/SDL2-with-libdecor.json
```

Comment:
this is in the 23.08 runtime.

Reply:
Nice pointer :smile: removed and tested, it actually works :+1: